site/documentation/entry.js

```diff
@@ -14,7 +14,7 @@ class Entry extends React.Component {
     const { props } = this;
 
     const example = !props.parsedComment.example ? null : (
-      <HtmlExample code={props.parsedComment.example.description} />
+      <HtmlExample code={props.parsedComment.example.description} nocopy={true} />
```

Comment:
Why not make this `copy={false}`? I find that easier to understand than the double-negative (i.e. if I want copy it should be `nocopy={false}`).